.env.example

```
@@ -235,11 +235,12 @@ AURORA_CACHE_TOKEN_IN_REDIS=false
 #            prefers App tokens and falls back to OAuth.
 GITHUB_AUTH_MODE=app
 
-# GitHub App. See server/connectors/github_connector/SETUP_GITHUB_APP.md or run
-# `python3 server/scripts/register_github_app.py --org <your-org>` to bootstrap.
+# GitHub App. Setup guide: website/docs/integrations/connectors.md (GitHub section).
 GITHUB_APP_ID=
 GITHUB_APP_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_APP_CLIENT_ID key should go before the GITHUB_APP_ID key

(UnorderedKey)